tenable/io/v3/connectors/api.py

```diff
@@ -0,0 +1,389 @@
+'''
+Connectors
+==========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`connectors <connectors>` API endpoints.
+
+Methods available on `` tio.v3.connectors``:
+
+.. rst-class:: hide-signature
+.. autoclass:: Connectors
+    :members:
+'''
+from typing import Dict, List, Optional, Tuple, Union
+from uuid import UUID
+
+from requests import Response
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.base.iterators.explore_iterator import (CSVChunkIterator,
+                                                           SearchIterator)
+from tenable.io.v3.connectors.schema import (ConnectorCreate,
+                                             ConnectorListTrails)
+from tenable.utils import dict_clean, dict_merge
+
+
+class ConnectorsAPI(ExploreBaseEndpoint):
+    '''
+    This class contains methods related to Connectors API
+    '''
+
+    _path = 'api/v3/connectors'
+    _conv_json = True
+
+    def create(self,
+               name: str,
+               con_type: str,
+               network_id: Optional[str] = None,
+               schedule: Optional[dict] = None,
+               **params: dict
+               ) -> Dict:
+        '''
+        Create a connector.
+
+        :devportal:`connectors: create <connectors-create>`
+
+        Args:
+            name:
+                The name of the connector to create.
+            con_type:
+                The type of connector to create.
+                Example:
+                    'aws_keyless'
+            schedule:
+                The data import schedule
+                Example:
+                    ('days', 2)
+            network_id:
```

Comment:
Please add type in params in docstrings?
i.e name (str)
con_type (str)
schedule (dict, optional)
network_id (str, optional)

Reply:
are we adding type params at both places? In function constructor and in docstring?

Reply:
Yes